docker/README.md

```diff
@@ -0,0 +1,105 @@
+
+## [Devito] images
+
+Devito provides three main images that are targeting different architectures and/or using different compilers. In the following, all images are described as `imagename-*`. The `*` corresponds to, and should be swapped for, the different release tags `dev`, `latest` or `vx.x.x` depending on if you are interested in specific versions (`vx.x.x`), the latest stable release(`latest`), or the latest development status (`dev`)
+
+### [Devito] on CPU
+
+We provide two CPU images:
+- `devito:gcc-*` with the standard GNU gcc compiler.
+- `devito:icc-*` with the intel C compiler for intel architectures.
+
+these base images provide a working [Devito] environment for any CPU architecture and come with [Devito], `gcc/icc` and `mpi` preinstalled as well as utilities such as `jupyter` for usability and exploration of the package.
+
+To run this image locally, you will need `docker` to be installed. Once available, the following commands will get you started:
+
+```bash
+docker run --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 devitocodes/devito:gcc-latest
+docker run --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 --device=/dev/infiniband/uverbs0 --device=/dev/infiniband/rdma_cm devitocodes/devito:gcc-latest
+```
+
+or to run in user context on a cluster with shared filesystem, you can add the correct user config as docker options e.g.:
+
+```bash
+docker run --rm -it -v `pwd`:`pwd` -w `pwd` -u $(id -u):$(id -g) devitocodes/devito:gcc-latest python examples/seismic/acoustic/acoustic_example.py
+```
+
+**Notes**:
+In addition, the following legacy tags are available:
+
+- `devito:cpu-*` that corresponds to `devito:gcc-*`
+
+
+### [Devito] on GPU
+
+Second, we provide three types of images to run [Devito] on GPUs. These two images are tagged `devito:nvidia-nvc-*`, `devito:nvidia-clang-*`, and `devito:amd-*`.
+
+- `devito:nvidia-nvc-*` is intended to be used on NVIDIA GPUs. It comes with the configuration to use the `nvc` compiler for `openacc` offloading. This image also comes with cuda-aware MPI for multi-gpu deployment.
+- `devito:nvidia-clang-*` is intended to be used on NVIDIA GPUs. It comes with the configuration to use the `clang` compiler for `openmp` offloading. This image also comes with cuda-aware MPI for multi-gpu deployment.
+- `devito:amd-*` is intended to be used on AMD GPUs. It comes with the configuration to use the `aoompcc` compiler for `openmp` offloading. This image also comes with gpu-aware MPI for multi-gpu deployment. Additionally, this image can be used on AMD CPUs as well since the Rocm compiler are preinstalled. You will need to modify `DEVITO_PLATFORM`  to `` at runtime to reflect this architecture.
```

Comment:
same as above

also, this

"You will need to modify `DEVITO_PLATFORM` to `` at runtime to reflect this architecture"

can be dropped? or is it still needed?

Reply:
Still needed, is supposed to be `amd` fixing